tests/python/test_torch_io.py

```diff
@@ -156,7 +156,7 @@ def test_io_simple():
 
 @ti.torch_test
 def test_io_simple():
-    mat = ti.Matrix.field(2, 6, dt=ti.f32, shape=(), needs_grad=True)
+    mat = ti.Matrix.field(2, 6, dtype=ti.f32, shape=(), needs_grad=True)
```

Comment:
It seems we made a mistake in #1694 and it silently passed since torch is not tested in CI? I still think it would be great if we provide them in extension libraries like `taichi_torch` (#1686)... It's making main repo harder to maintain and test.

Reply:
Sorry for that mistake! I should have taken care these `dt=` more carefully.;P

At this moment doesn't our CI care about these `tests`? 

I felt confused WHY this mistake wasn't detected by the CI. If so, maybe we need to add tests into CI but not isolated them into another extension, because maybe providing them in another extension will make our users a little painful? and this IO functionality is a common use case,  in my quite personal opinion. ;-)

Reply:
> and this IO functionality is a common use case.

In fact, normal users almost always use `numpy` I/O instead.
`torch` I/O is only designed for ML people who needs less copy overhead when using `taichi` together with `torch`.

However, the `torch` package is so large and complicated that cannot be a hard-dep of `taichi`.
We can't ask all developers to install `torch` just for running all the tests.
So we invent `@ti.torch_test`, when `torch` is not installed, the test will be automatically skipped.

Reply:
Thanks for your kind explanation! Now I can fully get the point.